### PR TITLE
Additional English and future to-do

### PR DESCRIPTION
**Pull request by Oshratn**

### CODE REVIEW

File: src/content/basics/kubernetes-on-giant-swarm/index.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 +++
 title = "Details on Kubernetes on Giant Swarm"
 description = "Here you learn how we set up things for you and what we manage, so you don't have to."
-date = "2019-11-14"
+date = "2019-11-18"
 weight = 10
 type = "page"
 categories = ["basics"]
+last review date = "2019-11-18"
```
Comment (marians) [inline src/content/basics/kubernetes-on-giant-swarm/index.md]:
```suggestion
last_review_date = "2019-11-18"
```

Our site generator does not like to have spaces in front matter attribute names.

**Comment by marians** [comment]
@Oshratn If this was waiting for my approval, sorry. I missed that.

**Comment by Oshratn** [comment]
@marians now I can't merge this due to conflicts. Can you ping me so we can resolve this issue?

**Comment by marians** [comment]
@Oshratn I just resolved the conflict. Please don't forget your TODO comments in the bottom of the page.

Sorry for the log wait!

**Pull request merged**